benchutils/metrics.py

```diff
@@ -169,6 +169,26 @@ def auprc(observed, expected):
     return auprc_
 
 
+def kl_divergence(observed, expected):
+    """
+
+    Parameters
+    ----------
+    observed : np.array
+        The profile obtained by running a profiling method
+    expected : np.array
+        The ground truth relative abundance
+
+    Returns
+    -------
+    float
+        The KL divergence from the observed distribution to the expected
```

Comment:
Again I am very impressed by this metric. This can potentially be useful in my other project. Maybe spell out "Kullback-Leibler divergence" in the docstring?

Reply:
Great idea. Done.